index.js

```diff
@@ -553,6 +553,10 @@ PSPDFKitView.propTypes = {
    * Controls whether or not the document will be automatically saved. Defaults to automatically saving (false).
    */
   disableAutomaticSaving: PropTypes.bool,
+  /**
+   * Controls whether or not the document's forms are editable. Defaults to forms being editable (true).
+   */
+  enableFormEditing: PropTypes.bool,
```

Comment:
This leads to confusing API on Android where this property has higher importance than the `configuration.enableFormEditing`. Is this really needed for iOS?

Reply:
> This leads to confusing API on Android where this property has higher importance than the configuration.enableFormEditing (see my comment above #365 (files)).

I don't think we need to attribute priority to the prop over the configuration option. On iOS, the last used setting will override the previous one. See https://github.com/PSPDFKit/react-native/pull/365/files#r414619844

> Is this really needed for iOS?

No, but I think it's a nice to have and it does not impact things in any way that I can think of.